### PR TITLE
scrape.js: Search org for user

### PR DESCRIPTION
**Pull request by andrewda**
This searches the user's organization for their display name. If
the user is found, their username will be used.

Closes https://github.com/coala/gci-leaders/issues/8

### CODE REVIEW

File: lib/scrape.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const fs = require('fs')
 const json2yaml = require('json2yaml')
 
 const GH_USER_BASE = 'https://github.com/users'
+const GH_ORG_BASE = 'https://github.com/orgs' // Ignore InvalidLinkBear
```
Comment (jayvdb) [inline lib/scrape.js]: I think just add github.com to the ignore list in .coafile

File: lib/scrape.js [hunk 2]
```diff
@@ -155,14 +167,17 @@ async function findOrganization({
 async function findGitHubUser({ display_name }, org) {
   if (!org) return
 
-  display_name = display_name.replace(/ /g, '')
+  const shortName = display_name.replace(/ /g, '')
```
Comment (jayvdb) [inline lib/scrape.js]: spaces are not valid in github usernames, so this is a very basic "convert to valid username".  There was a npm package which checked legality of a name.  What we want is a smarter "convert name into most similar valid github probable username".  maybe a npm package already exists; if not, ... create one ;-)

**Review by jayvdb — CHANGES_REQUESTED** [review]

File: lib/scrape.js (hunk 2 above)
Comment (andrewda) [reply lib/scrape.js]: Oh man, alright, I'm running out of package naming ideas ;)

File: .coafile [hunk 3]
```diff
@@ -14,6 +14,7 @@ bears = LineLengthBear
 
 [all.links]
 bears = InvalidLinkBear
+link_ignore_regex = (github\.com|\{|\$)
```
Comment (blazeu) [inline .coafile:17]: There's `link_ignore_list` that uses globbing instead
`link_ignore_list = https://github.com/**`

Comment (blazeu) [reply .coafile:17]: ^ Optional.

**Review by jayvdb — APPROVED** [review]

**Comment by jayvdb** [comment]
ack https://github.com/coala/gci-leaders/pull/75/commits/f2934af1278f4087732774f655afc273534d0d27

**Comment by blazeu** [comment]
ack https://github.com/coala/gci-leaders/pull/75/commits/f2934af1278f4087732774f655afc273534d0d27

**Pull request merged**